### PR TITLE
Feat: update Tokyonight theme to Tokyonight Moon and add Tokyonight Day variant

### PR DESCRIPTION
**Pull request by echoriiku**
- Updated the Tokyonight theme to the Tokyonight Moon variant
- Added a new variant, Tokyonight Day, following the namescheme of the Tokyonight project
- I think this fixes https://github.com/streetwriters/notesnook-themes/issues/82

@aureliuspublius could you please review these changes and let me know if everything looks good? Your feedback would be greatly appreciated.

### CODE REVIEW

**Comment by echoriiku** [comment]
Is it failing because I changed the theme folder name? It's using the correct namescheme now

**Review by thecodrr — CHANGES_REQUESTED** [review]
Why the formatting changes?

**Comment by echoriiku** [comment]
I was annoyed because my changes didn’t work at first. So, I threw the code into a `JSON` formatter, hoping it would fix some mistake I couldn’t find.

**Review by echoriiku — COMMENTED** [review]
Sorry for the unwanted changes, all good

**Review by echoriiku — COMMENTED** [review]
used the theme builder to correct the unintended formatting changes

File: themes/tokyonight-moon/v1/code-block.css
Comment (thecodrr) [inline themes/tokyonight-moon/v1/code-block.css:1]: I think you should run this through a formatter like https://www.cleancss.com/css-beautify/

File: themes/tokyonight-day/v1/code-block.css
Comment (thecodrr) [inline themes/tokyonight-day/v1/code-block.css:1]: I think you should run this through a formatter like https://www.cleancss.com/css-beautify/

**Review by thecodrr — CHANGES_REQUESTED** [review]

**Comment by thecodrr** [comment]
@echoriiku You can't change the id or the directory name for a theme. Since the id is internal, you can leave it be and simply change the title etc. If that doesn't work, you can just create a new theme with the new id.

**Review by echoriiku — COMMENTED** [review]
reverted the ID changes

**Review by thecodrr — APPROVED** [review]

**Pull request merged**